### DISCUSSION TITLE
Error during 'make PythonAPI'

### DISCUSSION THREAD

**Discussion by shuishenMath** [carla-simulator/carla#4215] [Q&A]
When I am trying to build CARLA on Windows, I got an error [during install_boost] Should I ignore this error message and continue to 'make launch'? 
![image](https://user-images.githubusercontent.com/74014715/118421438-37579f80-b68f-11eb-8e3f-5021790d8120.png)

**Comment by corkyw10** [comment]
Hi @shuishenMath, yes you can ignore that. We will be fixing this soon but in the meantime, the boost library is built with a backup that we have.

**Reply by shuishenMath** [reply]
Thank you!
